ingest-app/src/cmr/ingest/api/core.clj

```diff
@@ -233,6 +231,16 @@
   [concept]
   (pr-str (dissoc concept :metadata)))
 
+(defn log-concept-with-metadata-size
+  "Log the concept with metadata-size-bytes added, and metadata removed."
+  [concept request-context]
+  (let [metadata-size-bytes (-> (:metadata concept) (.getBytes "UTF-8") alength)
```

Comment:
Are we sure metadata is always UTF-8 encoded?

Reply:
Please refer to the description at the beginning of the PR.  @ygliuvt,  @chris-durbin, what do you guys think?

Reply:
There are probably many places in our code where we assume UTF-8. I think for the purposes of calculating bytes it is fine to assume UTF-8.